tests/integration_test/services_test.py

```
@@ -34148,6 +34148,97 @@ def test_add_one_time_schedule_integration_failure(mock_request_event_server):
     assert "Event server failed" in body["message"]
 
 
+@patch("kairon.shared.utils.Utility.request_event_server", autospec=True)
+def test_update_message_broadcast_one_time_scheduled_broadcast(mock_event_server):
```

Ruff (0.14.13)

Unused function argument: `mock_event_server`

(ARG001)